File: scenes/loading/Loading.cs
using Godot;

public class Loading : Base
{
    public override void _Ready()
    {
        base._Ready();
        GetNode<Timer>("Timer").Start();
    }

    public void OnTimerTimeout()
    {
        SwitchTo("res://scenes/menu/Menu.tscn");
    }
}

File: scenes/loading/Loading.fs
namespace BloodFeverFs.Scene

open KSGodotUtils
open Godot

type Loading() =
    inherit Base()

    override this._Ready() =
        base._Ready()
        this.getNode<Timer>("Timer").Start()

    member this.OnTimerTimeout() =
        this.SwitchTo("res://scenes/menu/Menu.tscn")
